source/features/clean-conversation-headers.css

```diff
@@ -30,7 +30,7 @@
 
 /* Removes on issues: [octocat] opened this issue on 1 Jan · 1 comments */
 /* Removes on PRs: [octocat] merged 1 commit into master from feature */
-.rgh-clean-conversation-headers.rgh-clean-conversation-headers-hide-author .author + .Label,
+.rgh-clean-conversation-headers.rgh-clean-conversation-headers-hide-author .author + .Label, /* #5832 */
```

Comment:
Related:

- #5832

Still used in:

- https://github.com/fregante/daily-version-action/issues/22